pom.xml

```diff
@@ -10,18 +10,23 @@
     <relativePath>./parent-pom.xml</relativePath>
   </parent>
 
-  <artifactId>snowflake-jdbc</artifactId>
+  <!-- Maven complains about using property here, but it makes install and deploy process easier to override final package names and localization -->

```

Comment:
Yeah, I agree with maven :) Maybe it would be better to use just `build.finalName`?

Reply:
It's not enough to change `finalName` and does not affect e.g. deploy plugin correctly